package.json

```diff
@@ -40,6 +40,8 @@
     "babelify": "^7.2.0",
     "bluebird": "^3.0.6",
     "browserify": "^12.0.1",
+    "lodash": "^3.10.1",
+    "tsify": "^0.13.1",
```

Comment:
I don't think `tsify` should be included by default. I also think that `babelify` could be removed, since this change makes no assumptions on default transforms. This will need some good docs / examples on common use case though.